Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs

```diff
@@ -43,6 +43,14 @@ public class ProfileSettings : ISoomlaSettings
 		static ProfileSettings()
 		{
 			SoomlaEditorScript.addSettings(instance);
+
+			List<string> additionalDependFiles = new List<string>(); //Add files that not tracked in file_list
+			additionalDependFiles.Add("Assets/Facebook");
```

Comment:
@Blinkq, I think we should remove there only `*.jar`/`*.a` only what was moved with https://github.com/vedi/unity3d-profile/blob/master/Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs#L78